client/src/components/growth-book/codeally-down.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { Alert } from '@freecodecamp/react-bootstrap';
+import { useFeature } from '@growthbook/growthbook-react';
+import Spacer from '../../components/helpers/spacer';
+
+export function CodeAllyDown(): JSX.Element | null {
+  const codeAllyDownFeature = useFeature('codeally_down');
+  const { t } = useTranslation();
+
+  if (!codeAllyDownFeature.on) {
+    return null;
+  }
+
+  return (
```

Comment:
```suggestion
  return codeAllyDownFeature.on ? (
```
A ternary cleans this up quite nicely.